### PR TITLE
Note Roslyn UsingTasks duplication in SDK

### PR DESCRIPTION
**Pull request by jjonescz**
https://github.com/dotnet/sdk/pull/41951#discussion_r1669378278

### CODE REVIEW

**Review by rainersigwald — APPROVED** [review]

**Comment by jjonescz** [comment]
It looks like I need a second review here and also someone to merge the PR, cc @rainersigwald

**Review by baronfel — APPROVED** [review]

**Pull request merged**
